### PR TITLE
Build cpptango 9.4.1

### PR DESCRIPTION
**Pull request by beenje**
Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [X] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [X] Ensured the license file is being packaged.

- bump version
- remove windows-external-symbol.patch

### CODE REVIEW

**Comment by beenje** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,20 +1,18 @@
...
-  sha256: 919daf8da36e3860150226c449d793f84cfe832ca0f133b9359bf2090312ac91
+  url: https://gitlab.com/tango-controls/{{ name }}/-/archive/{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 12f275fb5edc90a397655bec9a059215b5f7904c596c0f86270cbaa593de4f2a
   patches:
     - windows-lib-names.patch  # [win]
-    - windows-external-symbol.patch  # [win]
 
 build:
-  number: 1
+  number: 0
```
Comment (bourtemb) [inline recipe/meta.yaml:15]: I understood this build number needs to be incremented only for a new builds of the same version.
Here we are increasing the version so I would think we could keep it at 0.
See https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#build-number-and-string

Comment (beenje) [reply recipe/meta.yaml:15]: This is exactly what I did :-) I did reset it to `0`.

Comment (bourtemb) [reply recipe/meta.yaml:15]: :man_facepalming: Sorry for the noise @beenje

**Review by bourtemb — APPROVED** [review]
LGTM! Thanks @beenje

**Pull request merged**